firmware/python_modules/pixel/mpu6050.py

```diff
@@ -1,7 +1,9 @@
 import machine,utime,ustruct
```

Comment:
We hebben nu ook een C driver voor deze chip, want die zat op MCH2021 prototype 1 :smile: 
Misschien leuk om daar ooit naar toe te wisselen.

Reply:
Super nice, although specifically for CZ19 we need a modified driver (such as this python one) that has built-in retries and error checks, because of a hardware design issue (too weak pull-up values)